src/components/WalletConnect.vue

```diff
@@ -11,22 +11,40 @@
 						</a-menu>
 					</template>
 					<a-button>
-						{{ text }}
+						Connect Wallet
 						<DownOutlined />
 					</a-button>
 				</a-dropdown>
-				<a-button
-					v-if="selectedWallet !== WalletType.NONE"
-					type="primary"
-					@click="handleLogOut"
-				>
-					<template #icon>
-						<LogoutOutlined />
-					</template>
-				</a-button>
 			</a-col>
 		</a-row>
-	</div>
+	</template>
+	<template v-else>
+		<a-row class="wallet">
+			<a-col v-if="walletAddress">
+				<a-card size="small">
+					<a-row align="middle">
+						<a-col :span="20">
+							<a-descriptions :column="1" size="small" class="ellipsis">
+								<a-descriptions-item label="Wallet">{{
+									walletAddress
+								}}</a-descriptions-item>
+								<a-descriptions-item label="Selected Wallet">
+									{{ selectedWallet }}
+								</a-descriptions-item>
+							</a-descriptions>
+						</a-col>
+						<a-col style="margin: auto">
+							<a-button type="primary" @click="handleLogOut">
+								<template #icon>
+									<LogoutOutlined />
+								</template>
+							</a-button>
+						</a-col>
+					</a-row>
+				</a-card>
+			</a-col>
+		</a-row>
```

Comment:
do we need both row and col?

Reply:
No, since only one column is there, we can just have a row. will update